src/modules/shared/components/NavBar/index.js

```diff
@@ -0,0 +1,35 @@
+import { Link } from 'react-router-dom'
+import React from 'react'
+import TEDLogo from '../../../../common/images/TEDWhiteLogo.png'
+import styled from 'styled-components'
+
+const Nav = styled.nav`
+    background-color: rgba(0,0,0,0.5);
+`
+
+const NavLink = styled(Link)`
+    font-weight: bold;
+    color: white;
+    &:hover {
+        color: red;
+    }
+`
+
+const NavItem = styled.div`
+    display: flex;
+    flex-direction: row;
+`
+export default () => (
+  <Nav className='navbar fixed-top'>
```

Comment:
Use `sticky-top` instead of `fixed-top` so that the nav bar won't overlap the content.

Reply:
Changed !